blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -41,79 +50,176 @@ industry. We are committed to making a positive impact on the community
 through our work, and we look forward to continuing our open-source
 contributions in the future.
 
-## Our contributions:
+## Our contributions
 
-### coreboot
+![coreboot logo](/covers/coreboot-logo.svg)
 
-coreboot is an open-source firmware that provides a lightweight, secure,
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers)
+is an open-source firmware that provides a lightweight, secure,
 and fast boot experience for PCs, laptops, servers, and embedded devices.
+The changes made by our engineers can be divided into three main groups.
+
+#### Adding support for new platforms and maintaining existing ones
+
+Recently, our engineers have focused on expanding coreboot support for the
+latest platforms, it is worth mentioning that Karol Zmyslowski has added
+[support for Jasper Lake](https://review.coreboot.org/c/coreboot/+/73934).
+
+Michał Kopeć and Michał Żygowski had a major impact on Alder Lake support in
+coreboot. Michał Kopeć has added [support for Alder Lake](https://review.coreboot.org/c/coreboot/+/63374)
+and Michał Żygowski in the [set of patches](https://review.coreboot.org/c/coreboot/+/68449/11)
+has added improvements huge improvements. Also, Michał Żygowski made a significant
```

Comment:
`has added improvements huge improvements` - we have to be more precise, Michał added support that enables DMA protection, which is very important to protect against [DMA attacks](https://en.wikipedia.org/wiki/DMA_attack) at the firmware level. He also extended previously contributed [support for HSPHY](https://review.coreboot.org/plugins/gitiles/coreboot/+/9b0f169d2503dc044500e2790b987d766928782a) by making it work with DMA protection enabled.